website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -486,7 +488,8 @@ Also see the [advanced topics][advanced_topics] doc for more use cases and under
 [generating-crd]: https://book.kubebuilder.io/reference/generating-crd.html
 [markers]: https://book.kubebuilder.io/reference/markers.html
 [crd-markers]: https://book.kubebuilder.io/reference/markers/crd-validation.html
-[memcached_controller]: https://github.com/operator-framework/operator-sdk/blob/v1.2.0/testdata/go/memcached-operator/controllers/memcached_controller.go
+[memcached_controller]: https://github.com/operator-framework/operator-sdk/blob/v1.3.0/testdata/go/v3/memcached-operator/controllers/memcached_controller.go
+[memcached_controller-v2]: https://github.com/operator-framework/operator-sdk/blob/v1.2.0/testdata/go/memcached-operator/controllers/memcached_controller.go
```

Comment:
the tag would be the same. See that it has testdata/go/v3 and testdata/go/v2. So, it would be https://github.com/operator-framework/operator-sdk/blob/v1.3.0/testdata/go/v2/memcached-operator/controllers/memcached_controller.go instead. 

However, would we really encourage new users and new projects get done using the plugin go/v2 instead of the default one?

Reply:
My apologies, copy/paste!

I think you make a solid point about not needing or should v2 be mentioned. We would want users to use the default. Also, older versions of the doc can be referenced.

Reply:
Removed the v2 reference.